addons/account/models/account_move_line.py

```diff
@@ -1541,6 +1541,11 @@ def _search_panel_domain_image(self, field_name, domain, set_count=False, limit=
 
         # Override in order to not read the complete move line table and use the index instead
         query = self._search(domain, limit=1)
+
+        # if domain is logically equivalent to false
+        if not query:
```

Comment:
This is going to do a query to see if running the query has some results, before even adding `add_where` (so the memoize won't work)
```suggestion
        if not isinstance(query, Query):
```

(don't forget to import `from odoo.tools import Query`

Reply:
Sorry for the delay, the PR has been updated